### PR TITLE
refactor(breakoutRooms): migrate service to typescript

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Follow-up to #11649 
* TODO: Test coverage

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] ⛑️ Tests are included or not possible

### CODE REVIEW

File: src/stores/breakoutRooms.ts [hunk 1]
```diff
@@ -131,7 +131,8 @@ export const useBreakoutRoomsStore = defineStore('breakoutRooms', {
 				await this.fetchBreakoutRoomsParticipants(token)
 
 				// Open the sidebar and switch to the breakout rooms tab
-				emit('spreed:select-active-sidebar-tab', 'breakout-rooms')
+				// FIXME upstream https://github.com/nextcloud-libraries/nextcloud-event-bus/pull/734
+				emit('spreed:select-active-sidebar-tab', 'breakout-rooms' as unknown as object)
```
Comment (DorraJaouad) [inline src/stores/breakoutRooms.ts:135]: but it's not needed, it is always taken as a string.

**Review by DorraJaouad — APPROVED** [review]
Types checked and they are correct ✅

File: src/stores/breakoutRooms.ts (hunk 1 above)
Comment (Antreesy) [reply src/stores/breakoutRooms.ts:135]: TSC omplains here, as library don't expect a string yet

**Pull request merged**